jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointProperties.java

```diff
@@ -16,7 +16,8 @@
 
 package com.google.cloud.tools.jib.registry;
 
-/** Properties of registry endpoints. */
+/** Properties of registry endpoint requests. */
+// TODO: Probably rename to RegistryEndpointRequestProperties.
```

Comment:
Why not do this now?

Reply:
It requires lots of changes in other files (like tests) that I thought would be better to have in a separate PR to make this one simpler to review.